docs/content/en/docs/tutorials/exploring-metrics.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Exploring Tetragon Metrics"
+weight: 1
+icon: "overview"
+description: "Enable metrics on Tetragon and discover available metrics."
+---
+
+In this tutorial, we will deploy Tetragon as a standalone container (outside
+kubernetes) and a monitoring stack with Prometheus and Grafana to inspect metrics
+exposed by Tetragon.
+In the last part, we will also show how to access metrics when Tetragon is
+deployed in a Kubernetes cluster.
+
+### Explore metrics on the workstation with Docker
+
+#### requirements
+
+* Docker: either [Docker Engine](https://docs.docker.com/engine/install/) or [Docker Desktop](https://docs.docker.com/desktop/)
+* [Docker compose plugin](https://docs.docker.com/compose/install/)
```

Comment:
Hi, can you please split this to something simple 1. just docker container without compose and,  then 2. section 2 with compose and grafana stack so users can go from simple to advanced 

For the kubernetes part I guess we can drop it in favor of https://github.com/cilium/tetragon/pull/1495 unless there is good value for users then you can patch that section ;-)